client/dive-common/components/AttributesSubsection.vue

```diff
@@ -205,7 +210,7 @@ export default defineComponent({
               <v-icon small>
                 mdi-plus
               </v-icon>
-              Attribute
+              Add
```

Comment:
![image](https://user-images.githubusercontent.com/40494088/184271209-29007943-8fe2-473e-9ae8-4c311b192979.png)
I had something more like this in mind for the attribute headers. What do you think?